src/components/Initialization/Initialization.js

```diff
@@ -26,25 +26,40 @@ class Initialization extends PureComponent<Props, State> {
   };
 
   async componentDidMount() {
-    const nodeVersion = await getNodeJsVersion();
-
-    if (!nodeVersion) {
-      dialog.showErrorBox(
-        'Node missing',
-        'It looks like Node.js isn\'t installed. Node is required to use Guppy.\nWhen you click "OK", you\'ll be directed to instructions to download and install Node.'
-      );
-      shell.openExternal(
-        `${GUPPY_REPO_URL}/blob/master/README.md#installation`
-      );
-    }
+    try {
+      await initializePath();

```

Comment:
If this fails, we are not checking for node anymore and the app is in a state of loading indefinitely since the error is swallowed